pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rkbin";
+  version = "2023.07.26-${gitShort}";
```

Comment:
@samueldr do you have any comments on this? There really isn't any version number in upstream other than just the commit history. I believe that this is the best approach from what I know but I'd love to hear your thoughts!